flamingpy/utils/graph_states.py

```diff
@@ -17,93 +17,76 @@
 from flamingpy.codes.graphs import EGraph
 
 
-def star(n):
-    """EGraph of a star graph state with n nodes.
-
-    Args:
-        n (int): Number of qubits
-
-    Returns:
-        star_graph_state (EGraph)
-    """
-    assert isinstance(n, int), f"Input n should be an integer. Current type is {type(n)}"
-    assert n >= 2, f"Input n should be 2 or larger. Current value is {n}"
+def star_graph(n):
+    """Returns an EGraph of a star graph state with n nodes."""
+    if not isinstance(n, int):
+        raise ValueError(f"Input n should be an integer. Current type is {type(n)}")
+    if not n >= 2:
+        raise ValueError(f"Input n should be 2 or larger. Current value is {n}")
     star_graph_state = EGraph()
     for i in range(n - 1):
         degs = 2 * np.pi * i / (n - 1)
         x, y = np.cos(degs), np.sin(degs)
         edge = [(0, 0, 0), (x, y, 0)]
-        star_graph_state.add_edge(*edge, color="MidnightBlue")
+        star_graph_state.add_edge(*edge)
     return star_graph_state
 
 
-def ghz(n):
-    """EGraph of a GHZ state with n nodes.
-
-    Args:
-        n (int): Number of qubits
-
-    Returns:
-        ghz_state (EGraph)
-    """
-    assert isinstance(n, int), f"Input n should be an integer. Current type is {type(n)}"
-    assert n >= 3, f"Input n should be 3 or larger. Current value is {n}"
-    ghz_state = EGraph()
+def complete_graph(n):
+    """Returns an EGraph of a complete graph state with n nodes."""
+    if not isinstance(n, int):
+        raise ValueError(f"Input n should be an integer. Current type is {type(n)}")
+    if not n >= 3:
```

Comment:
```suggestion
    if n < 3:
```
Could we make this work for n = 1 (just a single node) and n = 2 (two nodes connected by an edge, e.g. a bell pair)?